airflow/opt/providers/etna/etna/plugins/impersonate_user_view.py

```diff
@@ -0,0 +1,54 @@
+from flask import flash
+
+from wtforms import SelectField
+from wtforms.validators import InputRequired
+from flask_appbuilder.fieldwidgets import Select2Widget
+from flask_appbuilder.forms import DynamicForm
+
+
+class ImpersonateUserForm(DynamicForm):
```

Comment:
Ok, cool.  So it is possible for us to use react in airflow if we need to  (build the static assets in and add them as a plugin resource), any "simple admin-y thing" would ideally be easy enough to idea with a basic python form.  This will make administration easier over time.

Reply:
Yes, we could totally use React components, etc., if we need to at some point (i.e. the REDCap config). `airflow-code-editor` uses a Vue.js front-end and compiles the assets, so we could do the same.